x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts

```diff
@@ -176,6 +176,10 @@ ${JSON.stringify(cypressConfigFile, null, 2)}
       const fleetServerPorts: number[] = [8220];
 
       const getEsPort = <T>(): T | number => {
+        if (isOpen) {
+          return 9220;
```

Comment:
nit: Avoid hardcoded values. Is there a chance the port changes and this file must be updated as well?

Reply:
nope, `9220` is the default port we are using in our FTR tests